Products/ZenUI3/browser/resources/js/zenoss/HierarchyTreePanel.js

```diff
@@ -518,7 +518,7 @@
                 if (keys[index] === undefined) {
                     return;
                 }
-                var node = current.findChild(field, keys[index]);
+                var node = current.findChild(field, keys[index], true);
```

Comment:
I'm not sure this will do the right thing in all cases.  The `expandNodeByPath` is already written to traverse down through the children.  Setting `deep` to `true` may cause a node to be returned that's not part of the requested path.